src/histolab/filters/compositions.py

```diff
@@ -47,6 +55,50 @@ def __new__(cls: type, cls_):
             f"Filters composition for the class {cls_.__name__} is not available"
         )
 
+    @lazyproperty
+    def tissue_mask_filters(self) -> imf.Compose:
+        """Return filters composition based on the ``cls_`` parameter.
+
+
+        Returns
+        -------
+        imf.Compose
+
+            If the ``cls_`` parameter is the class ``Slide`` the returned filters chain
+            is composed of:
+
+            - `image_filters.RgbToGrayscale()
+              <image_filters.html#src.histolab.filters.image_filters.RgbToGrayscale>`_
+
+            - `image_filters.OtsuThreshold()
+              <image_filters.html#src.histolab.filters.image_filters.OtsuThreshold>`_
+
+            - `morphological_filters.BinaryDilation()
+              <morphological_filters.html#src.histolab.filters.morphological_filters.BinaryDilation>`_
+
+            - `morphological_filters.RemoveSmallHoles()
+              <morphological_filters.html#src.histolab.filters.morphological_filters.RemoveSmallHoles>`_
+
+            - `morphological_filters.RemoveSmallObjects()
+              <morphological_filters.html#src.histolab.filters.morphological_filters.RemoveSmallObjects>`_
+
+            If the ``cls_`` parameter is the class ``Tile`` the returned filters chain
+            is composed of:
+
+            - `image_filters.RgbToGrayscale()
+              <image_filters.html#src.histolab.filters.image_filters.RgbToGrayscale>`_
+
+            - `image_filters.OtsuThreshold()
+              <image_filters.html#src.histolab.filters.image_filters.OtsuThreshold>`_
+
+            - `morphological_filters.BinaryDilation()
+              <morphological_filters.html#src.histolab.filters.morphological_filters.BinaryDilation>`_
+
+            - `morphological_filters.BinaryFillHoles(structure=np.ones((5, 5)))
+              <morphological_filters.html#src.histolab.filters.morphological_filters.BinaryFillHoles>`_
+        """
+        raise NotImplementedError
```

Comment:
I would use a meaningful message for this exception

Reply:
mmmm, this error will never never never be raised, because the `__new__` instantiates another class

Reply:
So we need it only for docstring?

Reply:
Yes exactly

Reply:
mmmm I never faced something like this ... but it seems weird to me.

Reply:
Mmm I never used the factory pattern in Python before 😅

Reply:
Actually even if the complexity is demanded to the factory ot other stupid methods, this docstring should be splitted into its derived methods in `TileComposition` and `SlideComposition` lazyproperties and get rid of this ugly thing

Reply:
Mmmm, but the classes are private and I guess the factory is used to hide the complexity.. how do we do this in practice?

Reply:
oh ok you right now i got it. But i would use a message anyway like: 
```
        raise NotImplementedError(
            "must be implemented by each subclass"
        )  # pragma: no cover
        ```

Reply:
alright coming